IpmiFeaturePkg/GenericIpmi/Common/GenericIpmi.h

```diff
@@ -6,8 +6,8 @@
   SPDX-License-Identifier: BSD-2-Clause-Patent
 **/
 
-#ifndef _IPMI_BMC_H_
-#define _IPMI_BMC_H_
+#ifndef _GENERIC_IPMI_H_

```

Comment:
nit: we normally make it as:
```suggestion
#ifndef GENERIC_IPMI_H_
```